README.md

```diff
@@ -2,69 +2,58 @@
-
-> This API documentation is subject to change in each Kubeapps version. If you are interested in the Kubeapps API, please [reach out to us](https://kubernetes.slack.com/messages/kubeapps) for further information.
+- [Kubeapps user guide](./docs/user/dashboard.md) to easily manage your applications running in your cluster.
+- [Kubeapps build guide](./docs/developer/build.md) for setting up the build environment and building Kubeapps from source.
+- [Kubeapps developer guide](./docs/developer/README.md) for setting up the developer environment.
+- [Kubeapps architecture](./docs/architecture/overview.md)
+- [Kubeapps FAQs](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/README.md#faq).
 
-## Differences from Monocular
+See how to deploy and configure [Kubeapps on VMware Tanzu™ Kubernetes Grid™](./docs/step-by-step/kubeapps-on-tkg/README.md)
 
-The [Monocular](https://github.com/helm/monocular) project was designed to run a public search and discovery website for Helm repositories (e.g. https://hub.kubeapps.com). Following its 1.0 release, Monocular is focused on delivering the experience for the Helm Hub.
+## Troubleshooting
 
-Versions of Monocular 0.7 and older include a basic ability to install, view and delete Helm releases in a Kubernetes cluster. To focus on the Helm Hub experience, these [features have been removed](https://github.com/helm/monocular#looking-for-an-in-cluster-application-management-ui) since the 1.0 release and are no longer supported. We recommend users of Monocular's in-cluster features try Kubeapps as it provides a more featured, robust and secure solution to managing Helm applications in your cluster.
+If you encounter issues, review the [troubleshooting docs](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/README.md#troubleshooting), [file an issue](https://github.com/kubeapps/kubeapps/issues), or talk to us on the [#Kubeapps channel](https://kubernetes.slack.com/messages/kubeapps) on the Kubernetes Slack server. 
 
-## Community
+- Click [here](http://slack.k8s.io) to sign up to the Kubernetes Slack org.
 
-- [#kubeapps on Kubernetes Slack](https://kubernetes.slack.com/messages/kubeapps)
+- Review our FAQs section on the [Kubeapps chart README](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/README.md#faq).
+## Contributing
 
-Click [here](http://slack.k8s.io) to sign up to the Kubernetes Slack org.
+If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](./CONTRIBUTING.md) documentation for guidance on how to setup Kubeapps for development.
 
-## FAQ
+## Changelog
 
-We maintain a section for frequently asked questions on the [Kubeapps chart README](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/README.md#faq).
+See the list of [releases](https://github.com/kubeapps/kubeapps/releases) to find out about feature changes.
```

Comment:
Not sure about the wording here, "_to find out about feature changes_" -> I'd change it to include also bug fixes (I mean, we also publish patch versions without new features).
So sth like

```suggestion
Take a look at the list of [releases](https://github.com/kubeapps/kubeapps/releases) to stay tuned for the latest features and changes.
```

But don't know; as it is the last sentence, we need a sort of a powerful finale